tools/repl/swift/REPLTime/REPLTime.h

```diff
@@ -0,0 +1,38 @@
+// REPLTime.h
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2017 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+// -----------------------------------------------------------------------------
+
+#ifndef REPL_TIME_H
+#define REPL_TIME_H
+
+typedef unsigned long long uint64_t;
```

Comment:
The repl_swift built purely by Xcode doesn't have debug information so we will not load it.
The answer here may simply be to make sure the binary is always stripped of its debug info.